aws/table_aws_vpc_flow_log.go

```diff
@@ -0,0 +1,227 @@
+				Description: "The ID of the VPC, subnet, or network interface.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "traffic_type",
+				Description: "The type of traffic (ACCEPT | REJECT | ALL).",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "log_destination_type",
+				Description: "Specifies the type of destination to which the flow log data is published.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "log_destination",
+				Description: "Specifies the destination to which the flow log data is published.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "log_format",
+				Description: "The format of the flow log record.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "max_aggregation_interval",
+				Description: "The maximum interval of time, in seconds, during which a flow of packets is captured and aggregated into a flow log record.",
+				Type:        proto.ColumnType_INT,
+			},
+			{
+				Name:        "tags_src",
+				Description: "A list of tags assigned to the VPC flowlog.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Tags"),
+			},
+
+			//standard columns
+			{
+				Name:        "tags",
+				Description: resourceInterfaceDescription("tags"),
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromP(getVpcFlowlogTurbotData, "Tags"),
+			},
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromP(getVpcFlowlogTurbotData, "Title"),
+			},
+			{
+				Name:        "akas",
+				Description: resourceInterfaceDescription("akas"),
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getVpcFlowlogAkas,
+				Transform:   transform.FromValue(),
+			},
+		}),
+	}
+}
+
+//// LIST FUNCTION
+
+func listVpcFlowlogs(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+
+	// TODO put me in helper function
+	var region string
+	matrixRegion := plugin.GetMatrixItem(ctx)[matrixKeyRegion]
+	if matrixRegion != nil {
+		region = matrixRegion.(string)
+	}
+	plugin.Logger(ctx).Trace("listVpcFlowlogs", "AWS_REGION", region)
+
+	// Create session
+	svc, err := Ec2Service(ctx, d, region)
+	if err != nil {
+		return nil, err
+	}
+
+	err = svc.DescribeFlowLogsPages(
+		&ec2.DescribeFlowLogsInput{},
+		func(page *ec2.DescribeFlowLogsOutput, lastPage bool) bool {
+			for _, item := range page.FlowLogs {
+				d.StreamListItem(ctx, item)
+			}
+			return !lastPage
+		},
+	)
+
+	return nil, err
+}
+
+//// HYDRATE FUNCTIONS
+
+func getVpcFlowlog(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	logger := plugin.Logger(ctx)
+	logger.Trace("getVpcFlowlog")
+	quals := d.KeyColumnQuals
+	flowlogID := quals["flow_log_id"].GetStringValue()
+	// TODO put me in helper function
```

Comment:
Add new line before comments
```suggestion

	// TODO put me in helper function
```